### ISSUE TITLE
Obsolete css rules override bootstrap behavior

### ISSUE THREAD

**Issue by EricSoroos** [ckan/ckan#4366]
These Less/CSS rules, added 6 years ago, appear to be blocking current bootstrap behavior. 

* https://github.com/ckan/ckan/blame/master/ckan/public/base/less/mixins.less#L134
* https://github.com/ckan/ckan/blame/master/ckan/public/base/less/mixins.less#L138

### CKAN Version if known (or site URL)

2.8

### Please describe the expected behaviour

Tab content shows on the page

### Please describe the actual behaviour

Tab content is hidden from the browser. 

### What steps can be taken to reproduce the issue? 

With a bootstrap tab hierarchy of:
```
<div class='tab-content'>
  <div class='tab-pane' />
  <div class='tab-pane active' />
</div>
```
None of the tab-panes will ever show, as the active class is applied to the tab panes, not the tab-content.  (ref: https://getbootstrap.com/docs/3.3/javascript/#markup)

**Comment by amercader** [comment]
Yes, this looks like it can be safely removed. @EricSoroos do you want to submit a quick PR doing so?

cc @gocemitevski

**Comment by gocemitevski** [comment]
@amercader is there an existing UI component in CKAN that is actually affected by this? Maybe tabs are not expected to work according to the Bootstrap behavior, when a parent element has the `.js` CSS class.

cc @EricSoroos

**Comment by amercader** [comment]
@gocemitevski Grepping the sources I only found `tab-content` being used in the stats plugin (without a `.js` parent) and no usage of `.js`

**Comment by gocemitevski** [comment]
@amercader in that case we cannot remove it until we re-implement the tabs navigation on stats. Here is how the page works on 2.6.6: https://demo.ckan.org/stats.

It is currently broken in 2.8.1 due to https://github.com/ckan/ckan/issues/4336.

cc @EricSoroos
